conf/DEV.public.conf

```diff
@@ -30,7 +24,19 @@ paypal {
         baseReturnUrl="https://contribute.thegulocal.com"
         paypalWebhookId="some_wehbhook_id"
     }
+
+}
+
+cors {
+    allowedOrigins = [
+        "https://support.thegulocal.com"
+    ]
+}
+
+support-frontend {
+    thank-you-url = "https://support.thegulocal.com/oneoff-contributions/thankyou"
```

Comment:
It would be nice to not have some of our config linked to the support frontend site - this could be a maintenance pain in the future. I'm wondering whether it's a security risk for you to be able to supply your own redirect URL in the CORS request? @adamnfish any thoughts?